enclave-manager/web/src/components/enclaves/modals/ConfigureEnclaveModal.tsx

```diff
@@ -155,6 +159,20 @@ export const ConfigureEnclaveModal = ({
   const handleLoadSubmit: SubmitHandler<ConfigureEnclaveForm> = async (formData) => {
     setError(undefined);
 
+    try {
+      console.debug("formData", formData);
+      if (formData.args && formData.args.args) {
+        formData.args.args = JSON.parse(formData.args.args);
+        console.debug("successfully parsed args as proper JSON", formData.args.args);
+      }
+    } catch (err) {
+      toast({
+        title: `An error occurred while preparing data for running package. The package arguments were not proper JSON: ${stringifyError("asdas")}`,
```

Comment:
I think the argument for the stringifyError is wrong, it should be stringifyError(err) I think

Reply:
oh good catch. Fixing!